app/forms/idv/api_image_upload_form.rb

```diff
@@ -353,6 +355,19 @@ def update_analytics(client_response:, vendor_request_time_in_ms:)
       )
     end
 
+    def birth_year(client_response)
+      dob = client_response.pii_from_doc&.dob
+      if dob.respond_to?(:year)
+        dob.year
+      else
+        begin
+          dob&.to_date&.year
+        rescue
+          nil
+        end
+      end
```

Comment:
Based on the code that parses responses from doc auth, `dob` should be `Date` or `nil` only, right? I think we can get rid of a lot of this extra handling

https://github.com/18F/identity-idp/blob/2c4137333fbe39baa57edf69738ae7ad102fbe82/app/services/doc_auth/lexis_nexis/doc_pii_reader.rb#L31-L36

```suggestion
      client_response.pii_from_doc&.dob&.year
```

Reply:
@zachmargolis , all mocking tests using string. I was converting that to `Date` in mock and not that clean.

I agree doing it here is not idea.